README.md

```diff
@@ -66,101 +66,7 @@ This is obviously the first diagnostic.  We try to provide helpful error message
-        type="System.Diagnostics.ConsoleTraceListener" 
-        initializeData="false"/>
-    </sharedListeners>
-    <trace autoflush="true">
-      <listeners>
-        <add name="console" />
-      </listeners>
-    </trace>    
-    <switches>
-      <add name="ADALLegacySwitch" value="Info"/>
-    </switches>
-  </system.diagnostics>
-```
-
-If you would like to have more control over how tracing is done in ADAL, you can add a `TraceListener` to ADAL's dedicated `TraceSource` with name **"Microsoft.IdentityModel.Clients.ActiveDirectory"**. 
-
-The following example shows how to write ADAL's traces to a text file using this method:
-
-```
-Stream logFile = File.Create("logFile.txt");
-AdalTrace.TraceSource.Listeners.Add(new TextWriterTraceListener(logFile));
-AdalTrace.TraceSource.Switch.Level = SourceLevels.Information;
-```
-
-You can achieve the same result by adding the following lines to your application's config file:
-
-```
-  <system.diagnostics>
-    <trace autoflush="true"/>
-    <sources>
-      <source name="Microsoft.IdentityModel.Clients.ActiveDirectory" 
-        switchName="sourceSwitch" 
-        switchType="System.Diagnostics.SourceSwitch">
-        <listeners>
-          <add name="textListener" 
-            type="System.Diagnostics.TextWriterTraceListener" 
-            initializeData="logFile.txt"/>
-          <remove name="Default" />
-        </listeners>
-      </source>
-    </sources>    
-    <switches>
-      <add name="sourceSwitch" value="Information"/>
-    </switches>
-  </system.diagnostics>
-``` 
-
-#### Windows Store Applications
-
-Tracing in ADAL for Windows Store is done via an instance of class `System.Diagnostics.Tracing.EventSource` with name **"Microsoft.IdentityModel.Clients.ActiveDirectory"**. You can define your own ```EventListener```, connect it to the event source and set your desired trace level. Here is an example:
-```
-var eventListener = new SampleEventListener();
-
-class SampleEventListener : EventListener
-{
-    protected override void OnEventSourceCreated(EventSource eventSource)
-    {
-        if (eventSource.Name == "Microsoft.IdentityModel.Clients.ActiveDirectory")
-        {
-            this.EnableEvents(eventSource, EventLevel.Verbose);
-        }
-    }
-
-    protected override void OnEventWritten(EventWrittenEventArgs eventData)
-    {
-	    ...
-    }
-}
-
-```
-
-There is also a default event listener which writes logs to a local file named **"AdalTraces.log"**. You can control the level of tracing to that event listener using the property ```AdalTrace.Level```. By default, trace level for this event listener is set to "None" and to enable tracing to this particular listener, you need to set the above property. This is an example:
-
-```
-AdalTrace.Level = AdalTraceLevel.Informational;
-```
+In order to configure logging, static property Callback of the LoggerCallbackHandler class should be set to the instance of class implementing IAdalLogCallback interface
```

Comment:
the instance of **a** class?
Also do we have such a class in our devApp? if yes, can we point to the line were LoggerCallbackHandler is set?

Reply:
yes we have it, but since it is straightforward how to set it and dev applications might be changed/ refactored I would not use line references.

Reply:
provided sample

Reply:
Great, thanks!